app/services/notifications/setup/submit-check.service.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+
+/**
+ * Orchestrates handling the data for `/notifications/setup/{sessionId}/check` page
+ * @module SubmitCheckService
+ */
+
+const SessionModel = require('../../../models/session.model.js')
+
+/**
+ * Orchestrates handling the data for `/notifications/setup/{sessionId}/check` page
+ *
+ * This service will transform the recipients into notification and start processing notifications.
+ *
+ * @param {string} sessionId - The UUID for the notification setup session record
+ *
+ * @returns {Promise<object>} The view data for the licence page
+ */
+async function go(sessionId) {
+  const session = await SessionModel.query().findById(sessionId)
+
+  // Temp - need event service pr to be merged
```

Comment:
need to add - https://github.com/DEFRA/water-abstraction-system/pull/1753

When merged